### PR TITLE
[bitnami/postgresql-ha] feat: allow user to specify pod-management-policy

### PR DESCRIPTION
**Pull request by DorBreger**
### Description of the change

A small change to replace the static podMangementPolicy: Parallel in the postgres statefulset
### Applicable issues

- fixes #

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/postgresql-ha/templates/postgresql/statefulset.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ metadata:
   {{- end }}
 spec:
   replicas: {{ .Values.postgresql.replicaCount }}
-  podManagementPolicy: Parallel
+  podManagementPolicy: {{ .Values.postgresql.podManagementPolicy | default "Parallel" }}
```
Comment (joancafom) [inline bitnami/postgresql-ha/templates/postgresql/statefulset.yaml:20]: If the default value in `values.yaml` is `Parallel`, I think that we can safely do the following:

```suggestion
  podManagementPolicy: {{ .Values.postgresql.podManagementPolicy | quote }}
```

**Review by joancafom — COMMENTED** [review]
Thanks for your PR! I added a small comment, could you please check?

**Comment by DorBreger** [comment]
> Thanks for your PR! I added a small comment, could you please check?

Looks good

**Review by joancafom — APPROVED** [review]
LGTM! Thanks for your changes 🚀

**Pull request merged**